www/app.py

```diff
@@ -99,56 +79,91 @@ class HeartBeatManager():
         The json-formatted information about the rover to send to rovercode-web.
```

Comment:
The `param` documentation needs to be updated for the new constructor parameters.